.env

```diff
@@ -120,3 +120,8 @@ GRAFANA_FARO_ENDPOINT=
 GRAFANA_OTLP_ENDPOINT=
 GRAFANA_INSTANCE_ID=
 GRAFANA_CLOUD_APIKEY=
+
+# Pyroscope
+GRAFANA_PYROSCOPE_ADDR=http://host.docker.internal:4040
```

Comment:
Do we maybe want to make these configurable, like it's done for the Prometheus host and port?

Reply:
Something like this?

```env
GRAFANA_PYROSCOPE_HOST=http://host.docker.internal
GRAFANA_PYROSCOPE_PORT=4040
GRAFANA_PYROSCOPE_ADDR=${GRAFANA_PYROSCOPE_HOST}:${GRAFANA_PYROSCOPE_PORT}
```

That makes sense. I added this change in https://github.com/grafana/opentelemetry-demo/pull/22/commits/1cceabfc8a1a0f6724dffa8567106408efe06a82.

It may make sense to add Pyroscope as a service in the Docker Compose configurations, but I avoided that for the time being because 
* I'm not sure what the precedent is adding new services (especially supporting services)